src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -165,12 +169,17 @@ def _match_offers_bids(self):
                 break
             elif (math.floor(offer.price/offer.energy)) <= clearing_rate and \
                     (clearing_energy - cumulative_traded_offers) >= offer.energy:
+                # TODO: Used the clearing_rate as the original_trade_rate for the offers, because
```

Comment:
I think I need a little bit of background of the difference between aggregated market and p2p market. I couldnt find a good explanation from uncle google. If you have a short explanation, it would be great, but i am also happy to discuss offline.

Reply:
It is a short explanation but I think it is better to discuss offline to clear any misunderstandings